camera_calibration/doc/index.rst

```diff
@@ -1,18 +1,138 @@
-camera_calibration
-==================
+Overview
+========
 
-The camera_calibration package contains a user-friendly calibration tool,
-cameracalibrator.  This tool uses the following Python classes, which
-conveniently hide some of the complexities of using OpenCV's calibration
-process and chessboard detection, and the details of constructing a ROS
-CameraInfo message.  These classes are documented here for people who
-need to extend or make a new calibration tool.
+This package uses OpenCV camera calibration, described
+`here <https://docs.opencv.org/2.4/modules/calib3d/doc/camera_calibration_and_3d_reconstruction.html>`_.
```

Comment:
Use a more update version of the opencv documentation https://docs.opencv.org/4.x/d9/d0c/group__calib3d.html ?